datafusion/expr/src/expr.rs

```diff
@@ -1219,8 +1226,8 @@ impl fmt::Display for Expr {
                     write!(f, " NULLS LAST")
                 }
             }
-            Expr::ScalarFunction(ScalarFunction { func_def, args }) => {
-                fmt_function(f, func_def.name(), false, args, true)
+            Expr::ScalarFunction(fun) => {
```

Comment:
Doing so I think makes code like this easier / less inceptionish (Expr::ScalarFunction vs expr::ScalarFunction 🤯 )